oteapi/strategies/mapping/mapping.py

```diff
@@ -0,0 +1,48 @@
+"""Mapping filter strategy."""
+# pylint: disable=unused-argument
+from typing import TYPE_CHECKING
+
+from pydantic.dataclasses import dataclass
+
+from oteapi.models import MappingConfig, SessionUpdate
+
+if TYPE_CHECKING:  # pragma: no cover
+    from typing import Any, Dict, Optional
+
+
+@dataclass
+class MappingStrategy:
+    """Strategy for a mapping.
+
+    The mapping strategy simply adds more prefixes and triples to the
+    `prefixes` and `triples` fields in the session.
+
+    Nothing is returned to avoid deleting existing mappings.
+
+    **Registers strategies**:
+
+    - `("filterType", "mapping")`
```

Comment:
```suggestion
    - `("mappingType", "mapping/triples")`
```
Let's call it something more explanatory?

Reply:
Or simply:
```suggestion
    - `("mappingType", "triples")`
```